package.json

```diff
@@ -152,7 +152,7 @@
     "eslint-plugin-material-ui": "workspace:^",
     "eslint-plugin-mocha": "^10.4.3",
     "eslint-plugin-react": "^7.34.3",
-    "eslint-plugin-react-compiler": "0.0.0-experimental-51a85ea-20240601",
+    "eslint-plugin-react-compiler": "0.0.0",
```

Comment:
This version is wrong. It should be `0.0.0-experimental-0998c1e-20240625` (latest, see https://www.npmjs.com/package/eslint-plugin-react-compiler?activeTab=versions). I'll push the changes.

Reply:
Fixed https://github.com/mui/material-ui/pull/42792/commits/7f05946910dcd82eba338fb4a7414ac122b7d895